packages/core/src/useCheckbox/useCheckbox.ts

```diff
@@ -13,6 +13,8 @@ import { CheckboxGroupContext, CheckboxGroupKey } from './useCheckboxGroup';
 import { useFormField, exposeField, FormField } from '../useFormField';
 import { FieldTypePrefixes } from '../constants';
 import { useInputValidity } from '../validation';
+import { refreshInspector, registerCheckboxWithDevtools } from '@dev-tools';
```

Comment:
`registerCheckboxWithDevtools` sounds too specific, do we need special `registerXXXX` for each type? Can't that just be a string argument to show the tag since all fields at the moment have the exact interface.